components/css/global/_footer.scss

```diff
@@ -156,7 +156,25 @@
 					border-bottom:solid 1px #474747;
 				}
 				.panel-heading{
-					display:none;
```

Comment:
I just realized the footer has been updated here in the header branch. Do we want to revert this and make a new branch to discuss these updates or do them in this branch as well?

Reply:
Yes, I kept meaning to do that.

Reply:
It would probably make sense to have them in a separate branch.